src/zkevm_specs/state.py

```diff
@@ -377,6 +377,9 @@ def check_tx_log(row: Row, row_prev: Row):
     tx_id = row.id()
     pre_tx_id = row_prev.id()
     log_id = row.address()
+    prev_log_id = row_prev.address()
+    prev_tag = row_prev.tag()
+    tag = row.tag()
     pre_field_tag = row_prev.field_tag()
```

Comment:
```suggestion
    prev_tx_id = row_prev.id()
    log_id = row.address()
    prev_log_id = row_prev.address()
    prev_tag = row_prev.tag()
    tag = row.tag()
    prev_field_tag = row_prev.field_tag()
```

Use `prev_` everywhere for consistency.